### PR TITLE
Add mrnode 1.0.2

### PR DESCRIPTION
**Pull request by talonbragg**
Pull request for issue: #10638
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):https://github.com/talonbragg/Mr.Node
 * Official website (optional, not the repository):https://mrnode.tk
 * NPM package url (optional):https://npmjs.org/package/mrnode
 * License and its reference:MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 3
   - Count of stars: 13
   - Count of forks: 5
 * NPM download stats (optional):
   - Downloads in the last day: 186
   - Downloads in the last week: 598
   - Downloads in the last month: 1,508

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [ ] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/mrnode/1.0.2/package.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
+{
+  "name": "mrnode",
+  "version": "1.0.2",
+  "description": "A Javascript Library for Node.js Functions",
+  "main": "dist/js/main.js",
```
Comment (pvnr0082t) [inline ajax/libs/mrnode/1.0.2/package.json]: @talonbragg We don't need `"main"`, `"scripts"`, `"bugs"`, `"devDependencies"`, `"dependencies"`and `"directories"`. Let's remove it.

File: ajax/libs/mrnode/1.0.2/package.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    "Mr.Node",
+    "node"
+  ],
+  "author": "Talon Bragg <pvpgamecrash@gmail.com> (http://talonbragg.com)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/talonbragg/Mr.Node/issues"
+  },
+  "homepage": "https://www.mrnode.tk",
+  "autoupdate": {
```
Comment (pvnr0082t) [inline ajax/libs/mrnode/1.0.2/package.json]: There are 25 releases on NPM but 18 on GitHub repo. I think it's better to use NPM auto-update config instead. You can take a look at [auto-update.md](https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md).

File: ajax/libs/mrnode/1.0.2/package.json [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    "source": "git",
+    "target": "git://github.com/talonbragg/Mr.Node.git",
+    "basePath": "dist/js",
```
Comment (pvnr0082t) [inline ajax/libs/mrnode/1.0.2/package.json]: We can just use `"dist"` as basePath.

File: ajax/libs/mrnode/1.0.2/package.json [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    "files": [
+      "main.min.js",
+      "main.js"
```
Comment (pvnr0082t) [inline ajax/libs/mrnode/1.0.2/package.json]: Let's use `"**/*"` instead of these two. It will fetch all files under `dist` directory.

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by PeterDaveHello** [comment]
@sufuf3 please take this PR.

**Comment by PeterDaveHello** [comment]
Close due to no response.

**Pull request closed without merging**